packages/coccinelle/coccinelle.1.1.0/opam

```diff
@@ -26,17 +26,18 @@ depends: [
   "conf-python-3-dev"
   "conf-aclocal"
   "conf-autoconf"
-  "parmap"
+  "parmap" {>= "1.0-rc9"}
   "num"
 ]
+available: os != "macos"
```

Comment:
For posterity could you describe why this constraint has been added?

Reply:
I added this to pass the CI tests, I don't think that coccinelle officially supported on macos (at least I didn't find mentions about it). If it's too strict, I'll drop it.

Reply:
This seems in line with what was done in the 1.1.1 release: https://github.com/ocaml/opam-repository/pull/19762

Reply:
I also tested it locally and can confirm that it does not compile on macos:
```
.../.opam/4.13.1/bin/ocamlc.opt -unsafe -I parmap  -ccopt -D_GNU_SOURCE -ccopt -fPIC parmap/setcore_stubs.c -o parmap/setcore_stubs.o
parmap/setcore_stubs.c:47:17: error: implicit declaration of function 'thread_policy_set' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
      retcode = thread_policy_set(mach_thread_self(),
                ^
1 error generated.
make[2]: *** [parmap/setcore_stubs.o] Error 2
make[1]: *** [all] Error 2
make: *** [.../code/ocaml/coccinelle/bundles/parmap/parmap.cmxa] Error 2
```

Reply:
Incidentally, I can build parmap on mac, but it is probably a more recent version than the one vendored by coccinelle